client/src/App.js

```diff
@@ -4,8 +4,6 @@ import { Switch, Route } from 'react-router-dom';
 import LandingPage from './pages/LandingPage';
```

Comment:
I recommend using 'display: grid; place-items:center' to get the components in "shape" that they are shown in the middle of the screen.

Reply:
Thanks for your review.